beater/config/config.go

```diff
@@ -141,6 +142,18 @@ func NewConfig(version string, ucfg *common.Config, outputESCfg *common.Config)
 		return nil, err
 	}
 
+	if !c.Sampling.KeepNonSampled && !c.Aggregation.Enabled {
+		// Non-sampled transactions should only be dropped
+		// when transaction aggregation is enabled in the
+		// server. This means the aggregations performed
+		// by the APM UI will not have access to a complete
+		// representation of the latency distribution.
+		logger.Warn("" +
```

Comment:
this probably deserves some docs

Reply:
Will do, when I add docs - deferring them for now.